.github/workflows/spec.yml

```diff
@@ -0,0 +1,121 @@
+name: spec
+on:
+  push:
+  pull_request:
+    branches: [main, master]
+  schedule:
+    - cron: "0 6 * * 6" # Every Saturday 6 AM
```

Comment:
I think we can remove this, no real need for regularly scheduled CI-runs since it'll run frequently while under active development